From b5e4b0ca4384400cc8822ac06cdacbb97bc3c6e3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Klaus=20K=C3=A4mpf?= <kkaempf@suse.de>
Date: Tue, 3 Jul 2018 09:38:39 +0200
Subject: [PATCH 4/4] make home dir for tests

Author: Adam Majer <adam.majer@suse.de>
Upstream: https://tracker.mender.io/browse/CFE-2549
BNC#1016848
Summary: this tests requires home directory, otherwise spams logfile
---
 tests/load/run_lastseen_threaded_load.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tests/load/run_lastseen_threaded_load.sh b/tests/load/run_lastseen_threaded_load.sh
index 7dc5bcae7c25..60ea99af0c55 100755
--- a/tests/load/run_lastseen_threaded_load.sh
+++ b/tests/load/run_lastseen_threaded_load.sh
@@ -6,4 +6,6 @@ then
     exit 0;
 fi
 
+text -x ~/.cfagent/state || mkdir -p ~/.cfagent/state
+
 ./lastseen_threaded_load -c 1   4 1 1
-- 
2.17.1

